opentelekomcloud/data_source_opentelekomcloud_cce_cluster_v3.go

```diff
@@ -115,28 +155,55 @@ func dataSourceCCEClusterV3Read(d *schema.ResourceData, meta interface{}) error
 			" Please try a more specific search criteria")
 	}
 
-	Cluster := refinedClusters[0]
-
-	log.Printf("[DEBUG] Retrieved Clusters using given filter %s: %+v", Cluster.Metadata.Id, Cluster)
-
-	d.SetId(Cluster.Metadata.Id)
-
-	d.Set("name", Cluster.Metadata.Name)
-	d.Set("flavor_id", Cluster.Spec.Flavor)
-	d.Set("description", Cluster.Spec.Description)
-	d.Set("cluster_version", Cluster.Spec.Version)
-	d.Set("cluster_type", Cluster.Spec.Type)
-	d.Set("billing_mode", Cluster.Spec.BillingMode)
-	d.Set("vpc_id", Cluster.Spec.HostNetwork.VpcId)
-	d.Set("subnet_id", Cluster.Spec.HostNetwork.SubnetId)
-	d.Set("highway_subnet_id", Cluster.Spec.HostNetwork.HighwaySubnet)
-	d.Set("container_network_cidr", Cluster.Spec.ContainerNetwork.Cidr)
-	d.Set("container_network_type", Cluster.Spec.ContainerNetwork.Mode)
-	d.Set("status", Cluster.Status.Phase)
-	d.Set("internal", Cluster.Status.Endpoints[0].Internal)
-	d.Set("external", Cluster.Status.Endpoints[0].External)
-	d.Set("external_otc", Cluster.Status.Endpoints[0].ExternalOTC)
+	cluster := refinedClusters[0]
+
+	log.Printf("[DEBUG] Retrieved Clusters using given filter %s: %+v", cluster.Metadata.Id, cluster)
+
+	d.SetId(cluster.Metadata.Id)
+
+	d.Set("name", cluster.Metadata.Name)
+	d.Set("flavor_id", cluster.Spec.Flavor)
+	d.Set("description", cluster.Spec.Description)
+	d.Set("cluster_version", cluster.Spec.Version)
+	d.Set("cluster_type", cluster.Spec.Type)
+	d.Set("billing_mode", cluster.Spec.BillingMode)
+	d.Set("vpc_id", cluster.Spec.HostNetwork.VpcId)
+	d.Set("subnet_id", cluster.Spec.HostNetwork.SubnetId)
+	d.Set("highway_subnet_id", cluster.Spec.HostNetwork.HighwaySubnet)
+	d.Set("container_network_cidr", cluster.Spec.ContainerNetwork.Cidr)
+	d.Set("container_network_type", cluster.Spec.ContainerNetwork.Mode)
+	d.Set("status", cluster.Status.Phase)
+	d.Set("internal", cluster.Status.Endpoints[0].Internal)
+	d.Set("external", cluster.Status.Endpoints[0].External)
+	d.Set("external_otc", cluster.Status.Endpoints[0].ExternalOTC)
 	d.Set("region", GetRegion(d, config))
 
+	cert, err := clusters.GetCert(cceClient, d.Id()).Extract()
```

Comment:
Do you use `cert` var?

Reply:
Yes, `cert.Clusters` and `cert.Users` later. And `go vet` should fail CI if var is unused